runestone/reveal/test/pavement.py

```diff
@@ -32,7 +32,8 @@
                        'use_services': 'false',
                        'python3': 'false',
                        'dburl': '',
-                       'basecourse': 'reveal-test'
+                       'basecourse': 'reveal-test',
+                       'dynamic_pages': True
```

Comment:
It would be nice to have two test cases, one with dynamic and other without.

Reply:
Ugh!

Reply:
Yes, that's a pain.

**But**, if we actually commit to two cases (static and dynamic), then part of the additional burden is writing those cases. (Not that I did this for the original dynamic book routing PR...ahem...). Fundamentally, supporting both static and dynamic versions of the site will be a lot of work, especially if we implement a lot of neat dynamic features.

Reply:
I think if we want to test all of these for both dynamic_pages == True / False then we should find a way to parameterize the pavement.py file at run time.  I think that would be a reasonable thing to do in a second PR so that books can be built multiple times with different settings.  

Maybe we need to update the runestone build command to allow for any of the template variables to be overwritten from the command line.

Also, this is not really a very good test of the dynamic mode as all it does is verify that the template if statement is there.  To really test the dynamic mode stuff we ought to do in the context of running in a docker container with a fully capable server.

Reply:
Good points -- I agree!